packages/stark-ui/src/modules/table/components/table.component.ts

```diff
@@ -864,22 +864,25 @@ export class StarkTableComponent extends AbstractStarkUiComponent implements OnI
 	 */
 	public onReorderChange(column: StarkColumnSortChangedOutput): void {
 		if (column.sortable) {
-			this.resetSorting(column);
 			const sortedColumn = find(this.columns, { name: column.name });
+			const newOrderProperties: string[] = [];
 			if (sortedColumn) {
 				sortedColumn.sortPriority = 1;
 				switch (column.sortDirection) {
 					case "asc":
-						sortedColumn.sortDirection = "desc";
+						newOrderProperties.push("-" + sortedColumn.name);
```

Comment:
Since there is no loop here (the method is called everytime the user clicks on the column name), could we change this to directly change the orderProperties value ?
`orderProperties = ["-" + sortedColumn.name]`

What is the benefit of having the const `newOrderProperties` ?